request.go

```diff
@@ -2097,6 +2264,23 @@ func (r *Request) setBody(setter string, reader io.Reader, len int, overwrite bo
 	r.bodySetter = setter
 }
 
+func (r *Request) checkOrder(funcCall string) bool {
+	if r.expectCalled {
+		r.chain.fail(r.unexpectedExpectError(funcCall))
+		return false
+	}
+	return true
+}
+
+func (r *Request) unexpectedExpectError(funcCall string) AssertionFailure {
```

Comment:
Since unexpectedExpectError() is used only in one place one, we can get rid of the function and inline it.